src/messages/initialize.cc

```diff
@@ -153,9 +153,9 @@ struct InitializeHandler : BaseMessageHandler<Ipc_InitializeRequest> {
       if (config->indexerCount == 0) {
         // If the user has not specified how many indexers to run, try to
         // guess an appropriate value. Default to 80% utilization.
-        const float kDefaultTargetUtilization = 0.8;
-        config->indexerCount =
-            std::thread::hardware_concurrency() * kDefaultTargetUtilization;
+        const float kDefaultTargetUtilization = 0.8f;
+        config->indexerCount = (int)(
```

Comment:
Did the old one cause `warning C4244: 'initializing': conversion from 'const float' to 'int', possible loss of data`?

Reply:
Yes. Actually there were two C4244 warnings: conversion from 'double' to 'float' and conversion from 'float' to `int`.